lib/test-worker.js

```diff
@@ -86,6 +86,9 @@ var oldNodeModulesPaths = module.constructor._nodeModulePaths;
 module.constructor._nodeModulePaths = function () {
 	var ret = oldNodeModulesPaths.apply(this, arguments);
 	ret.push(nodeModulesDir);
+	if (process.env.NODE_PATH) {
+		ret.push(process.env.NODE_PATH);
```

Comment:
`NODE_PATH` can contain multiple values. Not sure if the env var needs to be split and concatenated with `ret` here. See <https://nodejs.org/api/modules.html#modules_loading_from_the_global_folders> for more.